### ISSUE TITLE
Errors when installing HyperSpy bundle on virtual Windows 10

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy-bundle#50]
I installed the HyperSpy bundle (HyperSpy-bundle-2021.01.31-Windows-x86_64-AMD.exe) in a clean virtual Windows 10 machine (64-bit). During this processes, I encountered some issues:

- Trying to launch HyperSpy via Jupyter QtConsole gave an error about missing "VCRUNTIME140.dll", this was fixed by install "Microsoft Visual C++ Redistributable for Visual Studio": https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0
- Afterwards, starting "Jupyter qtconsole here": gives the error "No module named 'PySide' ... No Qt bindings could be found"
- "Jupyter lab here" and "Jupyter notebook here" briefly opens a command line window, which closes almost immediatly, without starting any browser.
- Starting the bundle python intepreter in `Users/USERNAME/HyperSpy-bundle`, `python.exe`, and running `import numpy as np` gives the error
     - `ImportError: DLL load failed while importing _multiarray_umath: The specified module could not be found. ... Importing the numpy C-extensions failed. This error can happen for many reasons, often due to issues with your setup or how NumPy was installed. ... The Python version is: Python3.8. The NumPy version is: "1.19.5"`.

**Comment by ericpre** [comment]
This is surprising, but it almost like the environment are not activated. Can you provide the output of `conda list` and `conda info`?

**Comment by ericpre** [comment]
I have just downloaded and installed it without any issue in a win10 virtual machine which had almost nothing installed in it. Was it single user or multiple user installation?

**Comment by magnunor** [comment]
I managed to narrow down the bug: it is (most likely) due to user account names with spaces in them. This results in the user home folder containing spaces. For example "C:\Users\test account".

Using a clean virtual machine, I made two new accounts. "magnunor" and "test account". Then I installed the HyperSpy bundle as mentioned above with for just that account. "Install for: Just me".

During the install procedure, it warns that spaces in the install path can lead to some conda packages breaking. However, installing Anaconda, then install hyperspy via conda-forge works fine.

To try to replicate this, you can create a new account via: Settings -> Accounts -> Family & other users -> Add someone else to this PC -> I don't have this person's sign-in information -> Add a user without a Microsoft account

**Comment by ericpre** [comment]
Indeed, using a path with a space in it is not recommended as warned during installation. For example, see these related issues: https://github.com/conda/constructor/issues/347 and https://github.com/ContinuumIO/anaconda-issues/issues/11794.

I can reproduce the issue with the context menu shortcut but the rest is working fine.

This is surprising that Anaconda and the HyperSpy bundle behave differently, because there are built in a very similar way. Have you tried the context menu with the Anaconda distribution with a space in the path?

**Comment by magnunor** [comment]
It seems indeed to be a bug with the context manager, rather than the bundle itself. As using the "Mini forge" -> "Mini prompt (HyperSpy bundle)" works fine with an account called "test account".

I tested the context manager in a clean Windows 10 Anaconda install, with user account (and path): "test account"

- Install Anaconda
- conda install hyperspy -c conda-forge
- conda install -c conda-forge start_jupyter_cm
- start_jupyter_cm # 

This gives the error: 

> 'C:\Users\test' is not a recognized as an internal or external command, operable program or batch file. 'jupyter-qtconsole.exe' is not recognized as an internal or external command operable program or batch file.

My guess is this is due to a bug somewhere in the context manager code.

Incidentally, in the install documentation in https://hyperspy.org/hyperspy-doc/current/user_guide/install.html, I didn't see anything about launching the HyperSpy bundle mini forge prompt via the "Mini Forge" start menu. So I didn't even notice it yesterday when trying to debug this, so it should maybe be mentioned somewhere in the install or getting started documentation.
